ocs_ci/ocs/ui/views.py

```diff
@@ -172,16 +197,22 @@
 }
 
 locators = {
+    "4.8": {
+        "login": login,
+        "page": page_nav,
+        "deployment": {**deployment, **deployment_4_7},
```

Comment:
just to be sure, is `**deployment_4_7` intentional here?

Reply:
yes. 
There is no difference between OCS4.8 and OCS4.7  installation [only MULTUS option]